controllers/bucket_controller.go

```diff
@@ -160,6 +160,7 @@ func (r *BucketReconciler) Reconcile(ctx context.Context, req ctrl.Request) (res
 				summarize.RecordReconcileReq,
 			),
 			summarize.WithResultBuilder(sreconcile.AlwaysRequeueResultBuilder{RequeueAfter: obj.GetInterval().Duration}),
+			summarize.WithPatchFieldOwner("source-controller"),
```

Comment:
Please map the controllerName const from main

Reply:
Thanks. I was wondering is there's already a constant for it.
Followed how it's done in kustomize-controller.